jazz_deployments/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "jazz_deployments",
+    "version": "0.0.1",
+    "description": "CRUD APIs for Deployments",
+    "main": "index.js",
+    "private": true,
+    "author": {
+        "name": "Jazz"
+    },
+    "scripts": {
+        "test": "mocha"
+    },
+    "dependencies": {
+        "request": "*",
+        "deasync": "*",
+        "async": "*",
+        "uuid": "*",
+        "json-query": "*",
+        "moment": "*",
+        "lodash": "*"
```

Comment:
i would not use lodash and async

Reply:
As the story indicates, we ned to remove async and deasync. 

Regarding lodash, if you are able to stay with native libraries w/o any helper code then switch to it and in the end if lodash isn't required, remove it from package.json.

Reply:
i agree and that is what i meant

Reply:
We will update this as part of next PR